qtodotxt/ui/controllers/main_controller.py

```diff
@@ -381,7 +381,13 @@ def revert(self):
     def openFileByName(self, filename):
         logger.debug('MainController.openFileByName called with filename="{}"'.format(filename))
         self._fileObserver.clear()
-        self._file.load(filename)
+        if not self._file.load(filename):
+            currentfile = self._settings.value("last_open_file", "")
+            if currentfile == filename:
+                self._dialogs.showError(self.tr("Current file '{}' was deleted".format(filename)))
+            else:
+                self._dialogs.showError(self.tr("Error opening file: {}".format(filename)))
```

Comment:
This is not C. Instead of adding a test here and trying to guess what happend, we should raise an explicative exception in _file.load()